src/insert-copy-filename/insert-copy-filename.js

```diff
@@ -47,6 +47,6 @@ export default function insertCopyFilename(diff: HTMLElement) {
     )
 
     const header: HTMLElement = (diff.querySelector('.filename'): any)
-    const lozenge: HTMLElement = (header.querySelector(':last-child'): any)
+    const lozenge: HTMLElement = (header.querySelector('h1 > :last-child'): any)
```

Comment:
Why this change ?

Reply:
When I was testing this last November, I found that with both the insert-copy-filename feature and this PR's feature enabled, the insert-copy-filename would assume that its button was the last one in the DOM, and could end up accidentally selecting the HTML for my button instead, causing issues.

I attempted to test this a few weeks ago, and could not replicate the issue, but would need to do more testing to feel confident one way or the other about removal.